.golangci.yml

```diff
@@ -162,9 +128,6 @@ linters-settings:
     sections:
       - standard
       - default
-      - prefix(github.com/cometbft)
-      - prefix(cosmossdk.io)
-      - prefix(github.com/cosmos)
       - prefix(github.com/provenance-io)
       - blank
       - dot
```

Comment:
Does this maybe need to go under `formatter.settings`? That might be why the linter doesn't seem to be applying these customizations.